SPECS/grub2/grub2.spec

```diff
@@ -60,6 +60,7 @@ BuildRequires:  systemd-devel
 BuildRequires:  xz-devel
 Requires:       device-mapper
 Requires:       xz
+Requires:       %{name}-configuration >= 2.06-12
```

Comment:
when did this -configuration package get added? i must have missed it...i don't see the point of having a separate -configuration package for the config files?

i guess we should keep it now for m2 (let's please drop it in azl3?), but for same-package deps like this you generally want to do a 1:1 requires, i.e. `Requires: %{name}-configuration = %{version}-%{release}`

Reply:
Because the grub2 package is "large" from M2's perspective. If you want to be able to install grub-configuring packages (kexec-tools) on a non-mkconfig system and then later have grub2-mkconfig work out of the box, you need these directories but not the tools. That is why we split grub2-configuration out.

Reply:
We could remove grub2-configuration for azl3 and enforce that images that are not mkconfig-based out-of-the-box will stay that way. Otherwise, I think we still need configuration and this change is relevant.

Reply:
https://github.com/microsoft/CBL-Mariner/pull/6299#discussion_r1340401013

Reply:
== %{version}